src/test/softtakeover_test.cpp

```diff
@@ -550,9 +546,9 @@ TEST_F(SoftTakeoverTest, PrevFarMore_NewFarMore_Late) {
     st_control.enable(co.data());
     
     // First is always ignored.
-    EXPECT_TRUE(st_control.ignore(co.data(), 120));
-    Time::setTestElapsedTime(SoftTakeover::TestAccess::getTimeThreshold()*2);
-    EXPECT_TRUE(st_control.ignore(co.data(), 100));
+    EXPECT_TRUE(st_control.ignore(co.data(), co->getParameterForValue(120)));
+    Time::setTestElapsedMsecs(SoftTakeover::TestAccess::getTimeThreshold()*2);
```

Comment:
optional -- style guide says to put spaces around * in multiplication: SoftTakeover::TestAccess::getTimeThreshold() * 2